src/http2_stream.erl

```diff
@@ -513,7 +502,7 @@ half_closed_local(
              Stream#stream_state{
                incoming_frames=queue:new(),
                response_body = Data
-              }};
+              }, 0};
```

Comment:
It lets me define something I want to happen on the transition to closed, so that no matter which state sends us there, it still happens on the `timeout` message